### PR TITLE
added breakpointPage and it is displayed at Q5

### PR DESCRIPTION
**Pull request by brkzmn**

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -68,8 +69,34 @@ const getCurrentQuestion = () => {
...
+  } else {
+    setTimeout(() => {
+      userInterfaceElement.innerHTML = '';
+      initGameOverPage()
+    }, 5000);
+  }
+
+
+  
+/*   if(e.target.previousElementSibling.id === currentQuestion.correct){
```
Comment (tomasvanrijsse) [inline src/pages/questionPage.js]: Is this part meant to be commented? I suppose it should be working

File: src/views/breakpointView.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    const element = document.createElement('div');
+    element.innerHTML = String.raw`
+    <img class="logo" src="../public/assets/logo_millionaire.webp" alt="game logo">
+    <p>You answered question ${index} correctly. You have reached breakpoint</p>
+    <p>Your current prize is ${prizes[(index)]} if you want, you can take this prize and leave.</p>
+    `
+    return element
+}
+/* <button class='btn btn-accept' id='${BREAKPOINT_BUTTON_ACCEPT_ID}'>Take the prize</button>
+<button class='btn btn-continue' id='${BREAKPOINT_BUTTON_CONTINUE_ID}'>Continue</button> */
```
Comment (tomasvanrijsse) [inline src/views/breakpointView.js]: How can the user proceed without these buttons?

File: src/data.js [hunk 3]
```diff
@@ -10,7 +10,7 @@
     not by your logic
     not by your listeners
 */
-export const prizes = ['100','200','500']
+export const prizes = ['100','200','500', '1000', '2000', '2000', '2000', '2000', '2000', '10000'];
```
Comment (tomasvanrijsse) [inline src/data.js]: This change seems not really correct or relevant for the current feature 🤷‍♂️

File: src/pages/questionPage.js (hunk 1 above)
Comment (brkzmn) [reply src/pages/questionPage.js]: We removed the comment. It is on the line 74.

File: src/views/breakpointView.js (hunk 2 above)
Comment (brkzmn) [reply src/views/breakpointView.js]: We added the buttons. They are working now.

File: src/data.js (hunk 3 above)
Comment (brkzmn) [reply src/data.js]: We updated the prizes. It can be also seen on the breakpointPage.

**Review by viveriveniversum — APPROVED** [review]

**Review by viveriveniversum — APPROVED** [review]

**Review by viveriveniversum — APPROVED** [review]

**Review by viveriveniversum — APPROVED** [review]

**Pull request merged**